src/android/GooglePlus.java

```diff
@@ -318,7 +326,21 @@ public void onActivityResult(int requestCode, final int resultCode, final Intent
      * @param signInResult - the GoogleSignInResult object retrieved in the onActivityResult method.
      */
     private void handleSignInResult(final GoogleSignInResult signInResult) {
+
+        if (this.safeMode) {
+            // savedCallbackContext was null in multiple crash reports, unable to reproduce, using this as fallback
+            // handle in cordova app e.g. timeouts 
+            if (savedCallbackContext == null) {
+                return;
```

Comment:
@alexp25 understood the concept here, but one thing if you pay attention is that you are returning from this bock if `savedCallbackContext` is `null`, and there is no callback for error and no other options to let the caller function know that there is an issue. 
Am asking this because `savedCallbackContext` itself is the callback and the method `error` is getting called in case of any error.
I doubt this can possibly create another issue which may lead to another potential crash?!